### PR TITLE
Improved RU translate

### PR DESCRIPTION
**Pull request by povar163**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

Closes #XXXXX

### CODE REVIEW

**Comment by greggubarev** [comment]
LGTM

File: curriculum/challenges/russian/06-information-security-and-quality-assurance/information-security-with-helmetjs/understand-bcrypt-hashes.russian.md [hunk 1]
```diff
@@ -8,23 +8,23 @@ localeTitle: Понимание
...
+Отправьте свою страницу, когда вы думаете, что вы поняли это правильно.
 </section>
 
 ## Tests
 <section id='tests'>
 
 ```yml
 tests:
-  - text: BCrypt is a dependency
+  - текст: BCrypt является зависимостью
```
Comment (RandellDawson) [inline curriculum/challenges/russian/06-information-security-and-quality-assurance/information-security-with-helmetjs/understand-bcrypt-hashes.russian.md]: @povar163 You should not be translating the yaml id
```suggestion
  - text: BCrypt является зависимостью
```

File: curriculum/challenges/russian/06-information-security-and-quality-assurance/information-security-with-helmetjs/understand-bcrypt-hashes.russian.md [hunk 2]
```diff
@@ -8,23 +8,23 @@ localeTitle: Понимание
...
     testString: getUserInput => $.get(getUserInput('url')+ '/_api/package.json').then(data => { var packJson = JSON.parse(data); assert.property(packJson.dependencies, 'bcrypt', 'Your project should list "bcrypt" as a dependency'); }, xhr => { throw new Error(xhr.statusText); })
-  - text: BCrypt has been properly required
+  - текст: BCrypt был должным образом необходим
```
Comment (RandellDawson) [inline curriculum/challenges/russian/06-information-security-and-quality-assurance/information-security-with-helmetjs/understand-bcrypt-hashes.russian.md]: @povar163 You should not be translating the yaml id
```suggestion
  - text: BCrypt был должным образом необходим
```

**Review by RandellDawson — CHANGES_REQUESTED** [review]
@povar163 See requested changes.

**Comment by RandellDawson** [comment]
Hi @povar163 

As there has been no response from your end for a while, we are closing this PR.

Feel free to request this PR to be reopened or create a new PR, if you still wish to make the changes you originally added plus any additional change requests asked of you.  Also, remember to update your fork, so you have the latest version of master in case it has changed significantly.

Happy Coding!

**Pull request closed without merging**